app/(main)/store/order-list/[id]/_components/order-list-dropdown/OrderListDropDown.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image';
+import { useState } from 'react';
+import polygon from '../../../../../../_assets/images/polygon.png';
+import reversePolygon from '../../../../../../_assets/images/reversePolygon.png';
+
+const OrderListDropDown = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [toggleMenu, setToggleMenu] = useState('전체');
+
+  const onClickToggleMenu = () => {
+    setIsMenuOpen(!isMenuOpen);
+  };
+
+  const onClickOption = (option: string) => {
+    setToggleMenu(option);
+    setIsMenuOpen(!isMenuOpen);
+  };
+
+  let buttonClasses = 'flex items-center gap-2 rounded text-xs';
+
+  if (toggleMenu === '전체') {
+    buttonClasses += ' text-black';
+  } else if (toggleMenu === '주문 접수') {
+    buttonClasses += ' text-green';
+  } else if (toggleMenu === '주문 확인') {
+    buttonClasses += ' text-orange';
+  } else if (toggleMenu === '주문 완료') {
+    buttonClasses += ' text-blue';
+  } else if (toggleMenu === '주문 취소') {
+    buttonClasses += ' text-red';
+  }
```

Comment:
해당 부분은 리팩토링할 부분이 보여서 한 번 건들어봤습니다..!
`tw-merge`를 설치했고 타입과 관련된 부분을 활용하면 좋을 것 같아요!

```suggestion
type DropDownText =
  | '전체'
  | '주문 접수'
  | '주문 확인'
  | '주문 완료'
  | '주문 취소';

const OrderListDropDown = () => {
  const [isMenuOpen, setIsMenuOpen] = useState(false);
  const [toggleMenu, setToggleMenu] = useState<DropDownText>('전체');
  ...
  
  const textColorConfigStyles = {
    전체: 'text-black',
    '주문 접수': 'text-green',
    '주문 확인': 'text-orange',
    '주문 완료': 'text-blue',
    '주문 취소': 'text-red',
  };

  const classNames = twMerge(
    `flex items-center gap-2 rounded text-xs ${textColorConfigStyles[toggleMenu]}`,
    추후에 props에서 커스텀하게 사용할 스타일 코드 추가 가능
  );
  ...
  
  return (
   <button onClick={onClickToggleMenu} className={classNames}>
   ...
```